src/ralph/back_office/models.py

```diff
@@ -84,6 +84,32 @@ def _check_user_assigned(instances):
     return errors
 
 
+def autocomplete_user_if_release_report(actions, objects):
```

Comment:
extract this logic to single function where you'll pass attribute name (owner, user)